openwisp_firmware_upgrader/upgraders/openwrt.py

```diff
@@ -155,8 +155,40 @@ def upload(self, image_file, remote_path):
         except Exception as e:
             raise RecoverableFailure(str(e))
 
+    def _verify_device_uuid(self):
+        """
+        Verifies that the UUID of the device being upgraded matches
+        the UUID in the device's configuration
+        """
+        device_uuid = self.upgrade_operation.device.pk
+        # Get UUID from device's openwisp config
+        output, exit_code = self.exec_command(
```

Comment:
Please add a test in which this method is mocked and returns a different UUID and verify that `UpgradeAborted` is raised as expected.

Reply:
I'll add